diff --git Makefile Makefile
index 0fb1f84..192474e 100644
--- Makefile
+++ Makefile
@@ -58,8 +58,8 @@ install-manpages: $(MANPAGES)
 
 install: install-mnexec install-manpages
 #	This seems to work on all pip versions
-	$(PYTHON) -m pip uninstall -y mininet || true
-	$(PYTHON) -m pip install .
+	$(PYTHON) -m pip uninstall -y mininet --break-system-packages || true
+	$(PYTHON) -m pip install . --break-system-packages
 
 develop: $(MNEXEC) $(MANPAGES)
 # 	Perhaps we should link these as well
diff --git util/install.sh util/install.sh
index bf4492d..ba17078 100755
--- util/install.sh
+++ util/install.sh
@@ -174,22 +174,27 @@ function mn_deps {
                         python-pep8 ${PYPKG}-pexpect ${PYPKG}-tk
     else  # Debian/Ubuntu
         pf=pyflakes
-        pep8=pep8
+        pep=pep8
         # Starting around 20.04, installing pyflakes instead of pyflakes3
         # causes Python 2 to be installed, which is exactly NOT what we want.
         if [ "$DIST" = "Ubuntu" -a `expr $RELEASE '>=' 20.04` = "1" ]; then
                 pf=pyflakes3
         fi
+        # Starting around 24.04, installing pycodestyle instead of pep8
+         if [ "$DIST" = "Ubuntu" ] &&  [ `expr $RELEASE '>=' 24.04` = "1" ]; then
+                pep=pycodestyle
+         fi
+
         # Debian 11 "bullseye" renamed
         # * pep8 to python3-pep8
         # * pyflakes to pyflakes3
         if [ "$DIST" = "Debian" -a `expr $RELEASE '>=' 11` = "1" ]; then
                 pf=pyflakes3
-                pep8=python3-pep8
+                pep=python3-pep8
         fi
 
         $install gcc make socat psmisc xterm ssh iperf telnet \
-                 ethtool help2man $pf pylint $pep8 \
+                 ethtool help2man $pf pylint $pep \
                  net-tools ${PYPKG}-tk
 
         # Install pip
@@ -203,7 +208,7 @@ function mn_deps {
             sudo ${PYTHON} get-pip.py
             rm get-pip.py
         fi
-       ${python} -m pip install pexpect
+       ${python} -m pip install pexpect --break-system-packages
         $install iproute2 || $install iproute
         $install cgroup-tools || $install cgroup-bin
         $install cgroupfs-mount
