diff --git bin/install-p4dev-v3.sh bin/install-p4dev-v3.sh
index a941954..40e568b 100755
--- bin/install-p4dev-v3.sh
+++ bin/install-p4dev-v3.sh
@@ -75,7 +75,6 @@ echo "  + thrift version 0.12.0 (not 0.9.2, because of a patch in this install s
 echo "  + nanomsg version 1.0.0"
 echo "  + nnpy git checkout c7e718a5173447c85182dc45f99e2abcf9cd4065 (latest as of 2015-Apr-22"
 echo "+ p4c: github.com/p4lang/p4c latest version"
-echo "+ Mininet: github.com/mininet/mininet latest version"
 echo "+ Python packages: grpcio 1.17.1, protobuf 3.6.1"
 echo "+ Python packages: crcmod, latest version"
 echo ""
@@ -436,27 +435,6 @@ date
 cd "${INSTALL_DIR}"
 find /usr/lib /usr/local $HOME/.local | sort > usr-local-6-after-p4c.txt
 
-set +x
-echo "------------------------------------------------------------"
-
-echo "Installing Mininet - not necessary to run P4 programs, but useful if"
-echo "you want to run tutorials from https://github.com/p4lang/tutorials"
-echo "repository."
-echo "start install mininet:"
-set -x
-date
-
-git clone git://github.com/mininet/mininet mininet
-sudo ./mininet/util/install.sh -nwv
-
-set +x
-echo "end install mininet:"
-set -x
-date
-
-cd "${INSTALL_DIR}"
-find /usr/lib /usr/local $HOME/.local | sort > usr-local-7-after-mininet-install.txt
-
 set +x
 echo "------------------------------------------------------------"
 echo "Installing a few miscellaneous packages"
@@ -517,8 +495,6 @@ diff usr-local-2-after-protobuf.txt usr-local-3-after-grpc.txt > usr-local-file-
 diff usr-local-3-after-grpc.txt usr-local-4-after-PI.txt > usr-local-file-changes-PI.txt
 diff usr-local-4-after-PI.txt usr-local-5-after-behavioral-model.txt > usr-local-file-changes-behavioral-model.txt
 diff usr-local-5-after-behavioral-model.txt usr-local-6-after-p4c.txt > usr-local-file-changes-p4c.txt
-diff usr-local-6-after-p4c.txt usr-local-7-after-mininet-install.txt > usr-local-file-changes-mininet-install.txt
-diff usr-local-7-after-mininet-install.txt usr-local-8-after-miscellaneous-install.txt > usr-local-file-changes-miscellaneous-install.txt
 
 P4GUIDE_BIN="${THIS_SCRIPT_DIR_ABSOLUTE}"
